src/components/views/elements/ImageView.js

```diff
@@ -65,6 +66,20 @@ module.exports = React.createClass({
         });
     },
 
+    getName: function () {
+        var name;
+
+        if(this.props.name) {
+            name = this.props.name;
+        } else if(this.props.mxEvent) {
+            name = props.mxEvent.getContent().body;
```

Comment:
heads up that the lack of `this.` on the `props` broke normal image loading... please be careful not to break existing functionality.  (and yes, I know we need better test coverage, but still...)